Sources/ScipioKit/Resolver/PackageResolver.swift

```diff
@@ -205,7 +207,7 @@ actor PackageResolver {
             modules: try await product.targets
                 .compactMap { targetName in manifest.targets.first(where: { $0.name == targetName }) }
                 .filter { shouldBuild($0.type) }
-                .asyncMap { try await self.resolve(target: $0, in: manifest) },
+                .asyncMap(numberOfConcurrentTasks: numberOfConcurrentTasks) { try await self.resolve(target: $0, in: manifest) },
```

Comment:
We discussed the issue of nested parallel execution.

While there are concerns, I believe it won't be a problem in normal Swift Concurrency, so it's sufficient to just keep it in mind.